contracts/SpokePool.sol

```diff
@@ -889,13 +919,54 @@ abstract contract SpokePool is
 
         USSRelayExecutionParams memory relayExecution = USSRelayExecutionParams({
             relay: relayData,
-            relayHash: bytes32(0),
+            relayHash: keccak256(abi.encode(relayData)),
```

Comment:
Was this previously overlooked or something? Surprised that tests passed with all relays having the same hash

Reply:
Nope if you look at the current `fillRelay` implementation, we first set this prop to 0 and then hash the entire RelayData struct. This is just a readability-motivated refactor since we pass in the `USSRelayData` struct into this function its easy to hash it in-place

Reply:
Oh I see!